src/http/server/request_processor.cr

```diff
@@ -47,7 +47,9 @@ class HTTP::Server::RequestProcessor
         context = Context.new(request, response)
 
         begin
```

Comment:
You can collapse the `begin ... end` pair into the newly added block.

Reply:
Yeah the exception should be logged inside the log context, so any context information set in a handler is applied to the debug message.